### DISCUSSION TITLE
Profile pictures not showing up in leaderboards & downloaded replays

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#16866] [Ideas] [answered]
When you download a replay and it puts it on the leaderboard, the profile picture doesn't show on the leaderboard on map select and at end of map. 

![image](https://user-images.githubusercontent.com/78441708/153802876-11a1c0d0-2067-4b29-8dda-66bca93dcaed.png)
![image](https://user-images.githubusercontent.com/78441708/153802959-6e44cc0f-5723-4ffa-a8a1-7f7a4d4f1611.png)

as you can see bubbleman has a profile picture
![image](https://user-images.githubusercontent.com/78441708/153802985-44f7910d-1b38-407a-b64a-f5aa46ccbec3.png)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/4315
